### DISCUSSION TITLE
LatinX is colonial language and not used by the people it is intended to describe.

### DISCUSSION THREAD

**Discussion by metaverde** [retextjs/retext#61] [General] [answered]
I was recently corrected for using the word Mexican to describe Mexican Indie Rock.
Mexican is a nationality, and is inclusive of Mexicans.
LatinX, in striving to solve the gender issues in the words Latino and Latina ignores the larger issue of describing Indigenous people in terms of their colonizers.

Additionally, [only about 3% of the people LatinX is meant to describe](https://www.washingtonpost.com/nation/2020/08/17/latinx-not-preferred-term-among-hispanics-survey-says/) use that language in describing themselves. 

Most Mexicans and South and Central Americans are happy to identify by their country, state, and sometimes tribe.
Many are Indigenous, speak no Spanish, and have no European ancestry, so Hispanic is not an inclusive term for people from Mexico, South, or Central America either.

I will make the additional point that nationalities such as Honduran and Guatemalan do not raise flags.
This is a very important issue and I trust it will soon be resolved.

**Comment by wooorm** [comment]
Could you describe how this relates to retext?

**Reply by Vyvy-vi** [reply]
Perhaps they are referring to this rule in [`retext-equality`](https://github.com/retextjs/retext-equality/):

![pic of word rule for Latinx](https://user-images.githubusercontent.com/62864373/106121797-f5f1c180-617d-11eb-999e-a54078426cb2.png)

**Comment by tvquizphd** [comment, marked as answer]
I've raised an issue [here](https://github.com/retextjs/retext-equality/issues/113) and found the right place to remove the suggestion to replace "Mexican" with "Latinx".
